tinylog-impl/src/main/java/org/tinylog/writers/AbstractFileBasedWriter.java

```diff
@@ -65,7 +65,7 @@ protected String getFileName() {
 	 *
 	 * @return Configured charset
 	 */
-	protected Charset getCharset() {
+	protected final Charset getCharset() {
```

Comment:
I think it is not necessary to make this method final. This could break other writers that are not part of tinylog itself.

Reply:
Updated.